custom-images/init.ps1

```diff
@@ -91,8 +95,8 @@ Set-EnvFileVariable "SITECORE_ID_CERTIFICATE" -Value (Get-SitecoreCertificateAsB
 Set-EnvFileVariable "SITECORE_ID_CERTIFICATE_PASSWORD" -Value $idCertPassword
 
 # SITECORE_LICENSE
-Set-EnvFileVariable "SITECORE_LICENSE" -Value (ConvertTo-CompressedBase64String -Path $LicenseXmlPath)
-
+#Set-EnvFileVariable "SITECORE_LICENSE" -Value (ConvertTo-CompressedBase64String -Path $LicenseXmlPath)
```

Comment:
@AmitKumar-AK What is the main reason for removing populating "SITECORE_LICENSE" for the .env file?

Reply:
I have configured a mounted license file instead of using SITECORE_LICENSE because it's started giving the error like `Server Error in '/' Application. 
Object reference not set to an instance of an object. 
Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code. 
Exception Details: System.NullReferenceException: Object reference not set to an instance of an object. 
Source Error: 
An unhandled exception was generated during the execution of the current web request. Information regarding the origin and location of the exception can be identified using the exception stack trace below. 
Stack Trace: 
[NullReferenceException: Object reference not set to an instance of an object.] 
   Sitecore.Nexus.Licensing.NexusLicenseApi.HasModule(String name) +146 `